services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -161,6 +160,26 @@ public Response getMainPage() throws IOException, FreeMarkerException {
         RequiredActionProviderModel updateEmailActionProvider = realm.getRequiredActionProviderByAlias(UserModel.RequiredAction.UPDATE_EMAIL.name());
         map.put("updateEmailActionEnabled", updateEmailActionProvider != null && updateEmailActionProvider.isEnabled());
 
+        final var devServerUrl = Environment.isDevMode() ? System.getenv(ViteManifest.ACCOUNT_VITE_URL) : null;
```

Comment:
I'm not sure we should be tying dev features to `start-dev`. From my perspective, `start-dev` is aiming to "Keycloak user who is developing a Keycloak deployment and wants loosen security" persona, not to "Keycloak developer" persona. We might even remove `start-dev` in the future as its name is confusing and replace it by something like `start --security-profile=dev`.

That said, I'm ok with this for now. As long as it's not documented in any releasable docs, we can change at any time.

Reply:
That's good to know, yeah this is entirely intended as a development only feature.

Reply:
@vmuzikar I'm curious to hear more about why `start-dev` is confusing and the plans to remove it. Do you have any issue about this?

In regards to the audience of the `start-dev` command, Keycloak Developers (core and community) are actually the people we have this command for, not only to loosen security but to also help on ow they develop their extensions on top of Keycloak. Even though this is something we don't fully achieved yet (e.g.: extension story), this command is for me the entry point for running the server for development and testing purposes.

Reply:
@pedroigor I think this was actually originally suggested by @stianst. Plan is to have "security profiles" (possibly even more than two – current prod and dev). It'd affect various defaults in KC, HTTP listener, client policies, initial admin user expiration, ... But that's future. :)

We might want a different mechanism for "developer profile".

Reply:
Thanks. I remember security profiles but IIRC they are different than the dev mode ...